srcpkgs/snixembed/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'snixembed'
+pkgname=snixembed
+version=0.2.2
+revision=1
+wrksrc="snixembed-${version}"
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="pkg-config vala"
+makedepends="gtk+3-devel glib-devel libdbusmenu-gtk3-devel libdbusmenu-glib-devel vala-devel"
+depends="libdbusmenu-gtk3 libdbusmenu-glib"
```

Comment:
If you need these `depends` just  because snixembed links against the libraries, you can leave them out of `depends` (and thus delete the whole line). `xbps-src` will figure out library dependencies automatically.